integrations/docker/images/chip-build-vscode/Dockerfile

```diff
@@ -75,7 +75,7 @@ ENV IDF_PATH=/opt/espressif/esp-idf/
 ENV IDF_TOOLS_PATH=/opt/espressif/tools
 ENV IMX_SDK_ROOT=/opt/fsl-imx-xwayland/5.10-hardknott/
 ENV NRF5_TOOLS_ROOT=/opt/NordicSemiconductor/nRF5_tools
-ENV NXP_K32W061_SDK_ROOT=/opt/sdk/sdks
+ENV NXP_K32W0_SDK_ROOT=/opt/sdk
```

Comment:
Changing in dockerfiles have to be created in a separate PR and then version has to be updated.

If we are building 0.5.87 today then the env is not the same and actually vscode image will NOT be a ble to build the current PR.

Reply:
I think we need this as a prerequisite so that vscode image can also compile things (all developers use vscode image)

Reply:
thanks, understood, will submit the changes from the vscode dockerfile in a separate PR.